src/BrowscapUpdater.php

```diff
@@ -43,102 +44,31 @@
      *
      * @var int
      */
-    private $connectTimeout = 5;
-
-    /**
-     * Gets a cache instance
-     *
-     * @return \BrowscapPHP\Cache\BrowscapCacheInterface
-     */
-    public function getCache() : BrowscapCacheInterface
-    {
-        if (null === $this->cache) {
-            $cacheDirectory = __DIR__ . '/../resources/';
-
-            $cacheAdapter = new File(
-                [File::DIR => $cacheDirectory]
-            );
-
-            $this->cache = new BrowscapCache($cacheAdapter);
-        }
-
-        return $this->cache;
-    }
-
-    /**
-     * Sets a cache instance
-     *
-     * @param \BrowscapPHP\Cache\BrowscapCacheInterface|\WurflCache\Adapter\AdapterInterface $cache
-     * @throws \BrowscapPHP\Exception
-     * @return self
-     */
-    public function setCache($cache) : self
-    {
-        if ($cache instanceof BrowscapCacheInterface) {
-            $this->cache = $cache;
-        } elseif ($cache instanceof AdapterInterface) {
-            $this->cache = new BrowscapCache($cache);
-        } else {
-            throw new Exception(
-                'the cache has to be an instance of \BrowscapPHP\Cache\BrowscapCacheInterface or '
-                . 'an instanceof of \WurflCache\Adapter\AdapterInterface',
-                Exception::CACHE_INCOMPATIBLE
-            );
-        }
-
-        return $this;
-    }
+    private $connectTimeout = self::TIMEOUT;
```

Comment:
Seems pointless to assign this, as it gets set in the constructor now anyway.

Reply:
done